src/main/java/com/iridium/iridiumskyblock/commands/CommandManager.java

```diff
@@ -107,7 +107,12 @@ public boolean onCommand(@NotNull CommandSender commandSender, org.bukkit.comman
                 User user = IridiumSkyblock.getInstance().getUserManager().getUser(player);
                 Optional<Island> island = user.getIsland();
                 if (island.isPresent()) {
-                    player.openInventory(new InventoryConfigGUI(IridiumSkyblock.getInstance().getInventories().islandMenu).getInventory());
+                    if (IridiumSkyblock.getInstance().getConfiguration().disableMainMenu) {
+                        String command = IridiumSkyblock.getInstance().getCommands().helpCommand.aliases.get(0);
```

Comment:
So I realised Im being really dumb when doing stuff this way, we should just do helpCommand.execute. Idk if u wana put another pr through to do that, if not ill do it